.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,14 @@
+name: Light House CI
+on: [push]
+jobs:
+  lighthouseci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+      - run: npm ci && npm install -g @lhci/cli@0.11.x
+      - run: lhci autorun
+        env:
+          LHCI_GITHUB_APP_TOKEN: ${{ secrets.LIGHTHOUSE_APP_TOKEN }}
```

Comment:
Where does this token come from? Where are the docs for this? And why do we need an extra token? And how can we get one?

Reply:
You can go through this link : https://github.com/apps/lighthouse-ci for setting up the token